app/controllers/application_controller.rb

```diff
@@ -2230,15 +2230,15 @@ def find_by_id_filtered(db, id)
                :model     => ui_lookup(:model => db.to_s)})
   end
 
-  def find_filtered(db, options = {})
+  def find_filtered(db)
     user     = current_user
     mfilters = user ? user.get_managed_filters : []
     bfilters = user ? user.get_belongsto_filters : []
 
     if db.respond_to?(:find_tags_by_grouping) && !mfilters.empty?
-      result = db.where(options[:conditions]).find_tags_by_grouping(mfilters, :ns => "*")
+      result = db.find_tags_by_grouping(mfilters, :ns => "*")
     else
-      result = db.apply_legacy_finder_options(options)
+      result = db
```

Comment:
Same question here...if the callers passed in extra conditions they would be lost here (Can I assume since you removed the signature that was never happening...did you verify with only tests, or did you click around or grep?)